plugins/home/api-report.md

```diff
@@ -66,8 +71,19 @@ export function createCardExtension<T>(options: {
   title: string;
   components: () => Promise<ComponentParts>;
   name?: string;
+  data?: {
+    homepageSettings?: HomepageSettings;
+  };
 }): Extension<(props: CardExtensionProps<T>) => JSX.Element>;
 
+// @public
+export const CustomHomepageGrid: (
+  props: {
+    children: ReactNode[];
+    config?: LayoutConfiguration[];
+  } & Omit<ResponsiveProps, 'layout'>,
```

Comment:
It would be good if we didn't have to export the types LayoutConfiguration and ResponsiveProps here at all. They pollute the API surface. The layout config should only ever come from the hook that fetches the current state from user settings, and hence not visible "outwards" to end users.